mne/_fiff/meas_info.py

```diff
@@ -2834,7 +2834,7 @@ def _read_extended_ch_info(chs, parent, fid):
             except KeyError:
                 # This shouldn't happen if we're up to date with the FIFF
                 # spec
-                warn(f"Discarding extra channel information kind {kind}")
+                warn("Discarding extra channel information kind %s" % (kind,))
```

Comment:
@drammock I think these we probably don't want to change... our `warn` function is really a wrapper around `warnings.warn` which doesn't use `*args` or delayed `%` formatting

Reply:
So we only want % for errors and for logger.info and logger.debug, but not for warn()?